hoomd/custom/custom_operation.py

```diff
@@ -54,13 +54,13 @@ def __init__(self, action, trigger=1):
     def __getattr__(self, attr):
         """Allows pass through to grab attributes/methods of the wrapped object.
         """
+        if attr == '_action':
+            raise AttributeError(
+                f"{type(self).__name__} object has no attribute _action")
```

Comment:
This is only called if `_action` isn't an attribute by the nature of `__getattr__`. This is to prevent infinite recursion in some edge cases where `_action` is not set.